obj/tasks.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    //DOM elements
+    createTaskButton : '[class="btn"]',
```

Comment:
this is a bad locator. What it does it press first button which have class "btn" there is no specific reference to this button which navigate user to create task

Reply:
checked the coded and it seems the best option to find it by this super generic class, so I take back the comment :D